google/cloud/bigtable/batcher.py

```diff
@@ -192,6 +192,10 @@ class MutationsBatcher(object):
     :type flush_interval: float
     :param flush_interval: (Optional) The interval (in seconds) between asynchronous flush.
         Default is 1 second.
+
+    :type flush_completed_callback: Callable = None
+    :param flush_completed_callback: (Optional) A callable funtion for handling responses 
```

Comment:
I dont think flush is correct here...from a user perspective it means all buffered elements have been sent, but that might not be the case. I think it should be batch_completed_callback\